chanfitness/chaneventstore.go

```diff
@@ -213,6 +227,10 @@ func (c *ChannelEventStore) Stop() {
 	// Stop the ticker after the goroutine reading from it has exited, to
 	// avoid a race.
 	c.cfg.FlapCountTicker.Stop()
+
+	log.Infof("ChannelEventStore shutdown complete")
```

Comment:
nit: I usually do an `info` level on `starting/stopping`, then a `debug` level log on `started/stopped` to prevent too many logs - not sure if it really helps since restart should be infrequent.

Reply:
Ok let's keep it like this then.